### ISSUE TITLE
*bug* Learning rate gets misreported to logging

### ISSUE THREAD

**Issue by cbockman** [tensorflow/tensor2tensor#688]
### Description

For non-legacy hparams.learning_rate_schedule, I believe learning rate is reported misleadingly:

https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/utils/t2t_model.py#L394

self.hparams.learning_rate is reported, whereas hparams.learning_rate_constant would seem more appropriate:

https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/utils/learning_rate.py#L30

Easy fix; happy to issue PR if I haven't misinterpreted things.

### *TensorFlow* and *tensor2tensor* versions

t2t master

**Comment by rsepassi** [comment]
Would welcome a PR. Thanks.

**Comment by rsepassi** [comment]
yes, thank you for reporting!
